chromadb/api/async_fastapi.py

```diff
@@ -230,12 +232,13 @@ async def create_collection(
             json={
                 "name": name,
                 "metadata": metadata,
+                "configuration": configuration.to_json() if configuration else None,
```

Comment:
Do we want this to be None? Should it be the default?

Reply:
I've made the segment responsible for constructing the default in the absence of the arg.